### PR TITLE
Clean up syntax for multihost_runner kill script

### PR DESCRIPTION
**Pull request by gobbleturk**
Philippe (pselo) recommended this cleaner syntax

### CODE REVIEW

File: multihost_runner.py [hunk 1]
```diff
@@ -133,29 +133,24 @@ def write_kill_script(script_dir, kill_processes_script_name):
...
-    echo "Found ${num_pids} unique PID(s) already running on your TPUs, killing now..."
-    for pid in "${pids_unique[@]}"; do
-        echo -e "Killing process ${pid}..."
-        kill -9 "${pid}"
-        tail --pid=$pid -f /dev/null
-        echo -e "Existing process ${pid} on your TPU was killed successfully!"
-    done
-    echo "All existing processes killed, so your TPU is ready to use!"
+echo "Searching for existing processes on device ${device_name}..."
+pids=$(sudo lsof -t /dev/${device_name}* | sort -u)
```
Comment (bvandermoon) [inline multihost_runner.py:142]: Was the -w option removed intentionally?

**Review by bvandermoon — APPROVED** [review]
LGTM, just a quick question

File: multihost_runner.py (hunk 1 above)
Comment (gobbleturk) [reply multihost_runner.py:142]: Ya Phillip thinks there is no reason for it. I believe -w suppresses warning messages, but I haven't seen any warning messages with or without -w in either case of a clean TPU or needing to kill PIDs

Comment (bvandermoon) [reply multihost_runner.py:142]: Great, thanks Matt

**Pull request merged**